module/TaskManagement/src/TaskManagement/Controller/RemindersController.php

```diff
@@ -81,6 +87,20 @@ public function create($data)
 					}
 				}
 				break;
+			case "add-estimation":
+				$task = $this->taskService->findTask (  $data['taskId'] );
+				if (is_null ( $task )) {
+					$this->response->setStatusCode ( 404 );
+					return $this->response;
+				}
+				
+				if (! $this->isAllowed ( $this->identity (), $task, 'TaskManagement.Task.sendReminder' )) {
```

Comment:
Il nome della risorsa deve riflettere la risorsa stessa, quindi in questo caso è meglio sia *TaskManagement.Reminder.add-estimation*

Sposta la verifica dell'accesso prima della *findTask* in modo che se l'utente non può accedere alla funzionalità il sistema non faccia alcuna operazione